test/README.md

```diff
@@ -0,0 +1,10 @@
+# Running Vault Helm Acceptance tests
+
+The Makefile at the top level of this repo contains a few target that should help with running acceptance tests in your own GKE instance.
+
+* Set the GOOGLE_CREDENTIALS and CLOUDSDK_CORE_PROJECT variables at the top of the file. GOOGLE_CREDENTIALS should contain the local path to your Google Cloud Platform account credentials in JSON format. CLOUDSDK_CORE_PROJECT should be set to the ID of your GCP project.
```

Comment:
Does the GOOGLE_CREDENTIALS file need to be in the current directory? For instance, it doesn't pick up my credentials file in test/terraform/:
```
$ make test-provision GOOGLE_CREDENTIALS=test/terraform/vault-helm-dev-creds.json CLOUDSDK_CORE_PROJECT=....
gcloud auth activate-service-account --key-file=test/terraform/vault-helm-dev-creds.json
ERROR: (gcloud.auth.activate-service-account) Unable to read file [test/terraform/vault-helm-dev-creds.json]: [Errno 2] No such file or directory: 'test/terraform/vault-helm-dev-creds.json'
make: *** [Makefile:56: provision-cluster] Error 1
make: *** [test-provision] Error 2
```
But it worked when I copied the credentials to my current directory.

Reply:
Hmmm that's strange. It seems to work ok for me:
make test-provision GOOGLE_CREDENTIALS=test/terraform/vault-helm-test.json
gcloud auth activate-service-account --key-file=test/terraform/vault-helm-test.json
Activated service account credentials for: [vault-terraform-helm-test@vault-helm-dev-246514.iam.gserviceaccount.com]
That's definitely the right name/path to the file?

Reply:
Oh sorry, I know what it was, `test/terraform/vault-helm-dev-creds.json` is a symlink outside the source tree in my setup. Nevermind 👍.